mulighetsrommet-api/src/main/resources/db/migration/V325__beregning_type.sql

```diff
@@ -0,0 +1,40 @@
+drop view if exists tilsagn_admin_dto_view;
+drop view if exists utbetaling_dto_view;
+
+create type tilsagn_beregning_type as enum (
+    'FRI',
+    'AVTALT_PRIS_PER_MANEDSVERK');
```

Comment:
Nitpick: Kunne kanskje hett bare 'PRIS_PER_MANEDSVERK' for å skille den fra prismodellen med samme navn. Eller et bedre forslag.